<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortefolioManagerClass</class>
 <widget class="QMainWindow" name="PortefolioManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PortefolioManager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PortefolioManager.qrc">
    <normaloff>:/PortefolioManager/Resources/images/icon.png</normaloff>:/PortefolioManager/Resources/images/icon.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::GroupedDragging</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="contentPlainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPreview">
    <property name="title">
     <string>Preview</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionToggle_Preview"/>
   </widget>
   <addaction name="menuWindows"/>
   <addaction name="menuPreview"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
  </widget>
  <widget class="QDockWidget" name="previewDockWidget">
   <property name="windowTitle">
    <string>Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="previewDockSubWidget">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWebEngineView" name="webEngineView">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="PortefolioManager.qrc">
     <normaloff>:/PortefolioManager/Resources/images/bold.png</normaloff>:/PortefolioManager/Resources/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="PortefolioManager.qrc">
     <normaloff>:/PortefolioManager/Resources/images/italic.png</normaloff>:/PortefolioManager/Resources/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="PortefolioManager.qrc">
     <normaloff>:/PortefolioManager/Resources/images/underline.png</normaloff>:/PortefolioManager/Resources/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionToggle_Preview">
   <property name="text">
    <string>Toggle Preview</string>
   </property>
   <property name="toolTip">
    <string>Toggle Preview</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PortefolioManager.qrc"/>
 </resources>
 <connections/>
</ui>
